### ISSUE TITLE
Can not run main_SCOPF.m

### ISSUE THREAD

**Issue by wto98769999** [CamilleH/Lecture-on-security-constrained-optimal-power-flow#1]
I try to run main_SCOPF.m  in MATLAB but it seems some important file is losing. I am wonder whether these files can be added in the project? Many thanks.

**Comment by abdelrahman-ayad** [comment]
Yeah it seems at least 2 functions are missing. @CamilleH Could you please help? Thanks.
